include/circt/Dialect/Seq/SeqOps.h

```diff
@@ -38,6 +39,28 @@ namespace seq {
 // HLMemType'd handle.
 bool isValidIndexValues(Value hlmemHandle, ValueRange addresses);
 
+/**
+ * Helper structure carrying information about FIR memory generated ops.
+ */
+struct FirMemory {
```

Comment:
nit: Please use `//` for multiple line comments as well (https://llvm.org/docs/CodingStandards.html#comment-formatting)